CODEOWNERS

```
@@ -460,6 +460,8 @@ build.json @home-assistant/supervisor
 /tests/components/hyperion/ @dermotduffy
 /homeassistant/components/ialarm/ @RyuzakiKK
 /tests/components/ialarm/ @RyuzakiKK
+/homeassistant/components/ialarmxr/* @bigmoby
+/tests/components/ialarmxr/* @bigmoby
```

```suggestion
/homeassistant/components/ialarmxr/ @bigmoby
/tests/components/ialarmxr/ @bigmoby
```

I guess the star can be omitted? I'm not sure about the allowed syntax of this file, but apparently all the other entries do no use it.